### PR TITLE
adding alt text to all images on code.org/hourofcode/overview

### PR DESCRIPTION
**Pull request by annaxuphoto**
This PR adds alt text to the images on code.org/hourofcode/overview. I verified all text added on localhost (aside from the strings added from the I18n G sheet).

I added 2 strings to the [I18n G sheet](https://docs.google.com/spreadsheets/d/1Tq7VqZALgRA0wYk0HDfEOTyRI0TM2Dir2rloXIPGCgU/edit?pli=1#gid=0), `top_hoc_tutorial_applab_alt_text` and `minecraft_tile_alt`

I'd love some feedback on the alt texts themselves! Any suggestions to make them more accessible would be appreciated.

Jira: https://codedotorg.atlassian.net/browse/PP-115
## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml [hunk 1]
```diff
@@ -94,6 +94,9 @@
   hoc_overview_csc_header: 'Introducing CS Connections'
   hoc_overview_csc_desc: 'Our newest curriculum highlights the connections between computer science and other subjects like math, language arts, science, and social studies. Through these modules, projects, and lessons, classrooms can explore their usual subjects in exciting new ways!'
   hoc_overview_csc_learn_more: 'Learn more about CS Connections'
+  hoc_danceparty_alt_text: 'gif of characters dancing underneath the text, Code your own dance party'
```
Comment (dju90) [inline pegasus/sites.v3/hourofcode.com/i18n/en.yml]: How about "Animated characters dancing to music underneath the text 'Code your own dance party'"

Comment (dju90) [reply pegasus/sites.v3/hourofcode.com/i18n/en.yml]: (following a best practice of not starting alt-text with "picture of..." or "image of..."

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml [hunk 2]
```diff
@@ -94,6 +94,9 @@
...
+  hoc_ai_alt_text: 'Illustration of AI bot'
```
Comment (dju90) [inline pegasus/sites.v3/hourofcode.com/i18n/en.yml]: How about "A floating, white, artificial intelligence character with an antennae, a scanner on the bottom, and an LED face"

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml [hunk 3]
```diff
@@ -94,6 +94,9 @@
...
+  hoc_banner_alt_text: 'Hour of Code banner'
```
Comment (dju90) [inline pegasus/sites.v3/hourofcode.com/i18n/en.yml:99]: Is this for the hoc banner for cs connections or for "more activities from partners"?

Comment (annaxuphoto) [reply pegasus/sites.v3/hourofcode.com/i18n/en.yml:99]: banner!

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml (hunk 1 above)
Comment (annaxuphoto) [reply pegasus/sites.v3/hourofcode.com/i18n/en.yml]: thanks for the suggestions! I'll update

File: pegasus/sites.v3/code.org/views/csc_promo.md.partial [hunk 4]
```diff
@@ -3,7 +3,7 @@
...
 
 ## Try out CS Connections with these activities for Hour of Code
-{{ tutorial_promo2, heading: "", background_image: "/images/fit-1940/csc/landing/tutorial_promo2_csc_background.jpg", left_heading: "Poem Art", left_description: "Express yourself! Use code to create animated poems.", left_specs: "", left_button_url: "https://studio.code.org/s/poem-art-2021/reset", left_button_text: "Start", right_heading: "Hello World", right_description: "Choose from four fun themes to code interactive characters in a world you create.", right_specs: "", right_button_url: "/helloworld", right_button_text: "Start"}}
+{{ tutorial_promo2, heading: "", background_image: "/images/fit-1940/csc/landing/tutorial_promo2_csc_background.jpg", alt_text: "Hour of Code banner", left_heading: "Poem Art", left_description: "Express yourself! Use code to create animated poems.", left_specs: "", left_button_url: "https://studio.code.org/s/poem-art-2021/reset", left_button_text: "Start", right_heading: "Hello World", right_description: "Choose from four fun themes to code interactive characters in a world you create.", right_specs: "", right_button_url: "/helloworld", right_button_text: "Start"}}
```
Comment (daynew) [inline pegasus/sites.v3/code.org/views/csc_promo.md.partial]: It's too bad the strings used in the partials are not translatable.

**Review by daynew — APPROVED** [review]
The descriptions looks good to me.

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml (hunk 3 above)
Comment (dju90) [reply pegasus/sites.v3/hourofcode.com/i18n/en.yml:99]: Cool - how about change this to: "The Hour of Code logo surrounded by a tiger, a smile emoji, a cupcake, an 8-bit blue worm, an octopus poetry instructor, and school subject icons representing math, language arts, biology, and art"

Comment (annaxuphoto) [reply pegasus/sites.v3/hourofcode.com/i18n/en.yml:99]: oh oops I misunderstood your question - this is for the "more activities from partners" image, but I'll update the banner text with your suggestion!

Comment (annaxuphoto) [reply pegasus/sites.v3/hourofcode.com/i18n/en.yml:99]: Note that the CS connections banner is not currently translatable

**Pull request merged**